docs/sources/setup-grafana/configure-docker.md

```diff
@@ -57,48 +91,217 @@ The following settings are hard-coded when launching the Grafana Docker containe
 | GF_PATHS_PLUGINS      | /var/lib/grafana/plugins  |
 | GF_PATHS_PROVISIONING | /etc/grafana/provisioning |
 
+## Install plugins in the Docker container
+
+You can install publicly available plugins and plugins that are private or used internally in an organization. For plugin installation instructions, refer to [Install plugins in the Docker container]({{< relref "./installation/docker#install-plugins-in-the-docker-container" >}}).
+
+### Install plugins from other sources
+
+To install plugins from other sources, you must define the custom URL and specify it immediately before the plugin name in the `GF_INSTALL_PLUGINS` environment variable: `GF_INSTALL_PLUGINS=<url to plugin zip>;<plugin name>`.
+
+Example:
+
+The following command runs Grafana Enterprise on **port 3000** in detached mode and installs the custom plugin, which is specified as a URL parameter in the `GF_INSTALL_PLUGINS` environment variable.
+
+```bash
+docker run -d
+  -p 3000:3000
+  --name=grafana
+  -e "GF_INSTALL_PLUGINS=http://plugin-domain.com/my-custom-plugin.zip;custom-plugin,grafana-clock-panel"
+  grafana/grafana-enterprise
+```
+
+## Build and run a Docker image with pre-installed plugins
+
+To create multiple images with the same plugins, you can save time by building your own customized image that includes those plugins.
+
+In the Grafana GitHub repository, the `packaging/docker/custom/` folder includes a `Dockerfile` that you can use to build a custom Grafana image. The `Dockerfile` accepts `GRAFANA_VERSION`, `GF_INSTALL_PLUGINS`, and `GF_INSTALL_IMAGE_RENDERER_PLUGIN` as build arguments.
+
+The `GRAFANA_VERSION` build argument must be a valid `grafana/grafana` Docker image tag. By default, Grafana builds an Alpine-based image. To build an Ubuntu-based image, append `-ubuntu` to the `GRAFANA_VERSION` build argument.
+
+Example:
+
+The following example shows you how to build and run a custom Grafana Docker image based on the latest official Ubuntu-based Grafana Docker image:
+
+```bash
+# go to the custom directory
+cd packaging/docker/custom
+
+# run the docker build command to build the image
+docker build
+  --build-arg "GRAFANA_VERSION=latest-ubuntu"
+  -t grafana-custom .
+
+# run the custom grafana container using docker run command
+docker run -d -p 3000:3000 --name=grafana grafana-custom
+```
+
+## Build a Grafana Docker image with pre-installed plugins
+
+To save time, you can customize a Grafana image by including plugins available on the [Grafana Plugin download page](/grafana/plugins). By doing so, you won't have to manually install the plugins each time, making the process more efficient.
+
```

Comment:
This is all duplicative with little benefit, I suggest we collapse this section into 1
```suggestion
```

Reply:
I spoke with @usmangt about this, and he's going to reach out to clarify what you want moved/combined.

Reply:
I have combined those 2 section and now availalbe as one under the heading "Build and run Grafana Docker image with pre-installed plugins"